config/flags_node.go

```diff
@@ -135,9 +135,21 @@ var (
 	// FlagTequilapiPort port for listening for incoming API requests.
 	FlagTequilapiPort = cli.IntFlag{
 		Name:  "tequilapi.port",
-		Usage: "Port for listening incoming api requests",
+		Usage: "Port for listening incoming Tequilapi requests",
```

Comment:
I'd prefer to keep it as `API requests` in the description. It's more clear for normal users who are not from the mysterium team. 
We already have `tequilapi` word in the flag name, the description should make it more clear and not adding extra confusion for users.

Reply:
Renamed descriptions for tequilapi.* flags